### PR TITLE
autocannon: add printResult definition

### PR DESCRIPTION
**Pull request by RafaelGSS**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## autocannon ([<kbd>unpkg</kbd>](https://unpkg.com/browse/autocannon@latest/))
was missing the following properties:
1. start
1. parseArguments
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 12b11735114cc6a910833a5f005af93030bbec25
</p>

File: types/autocannon/index.d.ts [hunk 1]
```diff
@@ -471,6 +471,10 @@ declare namespace autocannon {
      * Track the progress of your autocannon.
      */
     function track(instance: Instance, options?: TrackingOptions): void;
+    /**
+     * Return string to print the result tables to the terminal, programmatically.
+     */
+    function printResult(result: Result, options?: TrackingOptions): string;
```
Comment (joshuajeschek) [inline types/autocannon/index.d.ts:477]: I don't think that TrackingOptions is accurate here, if you take a look at the documentation, you can see that `renderProgressBar` and `progressBarString` are not listed as options for printResult:
[printResult Documentation](https://github.com/mcollina/autocannon#autocannonprintresultresultobject-opts)

**Review by joshuajeschek — CHANGES_REQUESTED** [review]

**Review by joshuajeschek — APPROVED** [review]
looking good now!

**Comment by RafaelGSS** [comment]
Ready to merge

**Pull request merged**